packages/cozy-harvest-lib/src/components/EditAccountModal.jsx

```diff
@@ -164,9 +164,9 @@ export class EditAccountModal extends Component {
   redirectToAccount() {
     const { account } = this.state
     if (account) {
-      this.props.pushHistory(`/accounts/${account._id}`)
+      this.props.replaceHistory(`/accounts/${account._id}`)
     } else {
-      this.props.pushHistory(`/accounts`)
+      this.props.replaceHistory(`/accounts`)
```

Comment:
I think here we should stay on pushHistory. Not replace. We edited an account, then we go to /accounts or /accounts/id. If I want to click on back, I should go to the edit modale again

Reply:
Updated ✅